specs/swagger.yaml

```diff
@@ -303,6 +390,21 @@ definitions:
         description: List of ontology terms providing attributes of the spectrum
         items:
           $ref: "#/definitions/OntologyTerm"
+  Psm:
+    required:
+      - accession
+      - peptideSequence
+    properties: 
+      accession:
```

Comment:
What exactly is meant by a PSM's "accession"? I don't remember talking about this.